src/modules/class-wpml-cornerstone-module-with-items.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Class WPML_Cornerstone_Module_With_Items
+ */
+abstract class WPML_Cornerstone_Module_With_Items implements IWPML_Page_Builders_Module {
+
+	/**
+	 * @param string $field
+	 *
+	 * @return string
+	 */
+	abstract protected function get_title( $field );
+
+	/** @return array */
+	protected function get_fields() {
+		return array( 'label', 'content' );
+	}
+
+	/**
+	 * @param string $field
+	 *
+	 * @return string
+	 */
+	protected function get_editor_type( $field ) {
+		switch ( $field ) {
+			case 'label':
+				return 'LINE';
+
+			case 'content':
+				return 'VISUAL';
+
+			default:
+				return '';
```

Comment:
Shouldn't you return a default value when we don't know the field? Maybe a line?

Reply:
@strategio oh this is useless. This is an abstract class, child classes will implement this. Let me fix that.